### ISSUE TITLE
[Feature Request] [tw_gcal_sync] Support virtual tags

### ISSUE THREAD

**Issue by amiroslaw** [bergercookie/syncall#59]
I've tried to synchronize tasks using virtual tag +DUE but I don't know if it was implemented. 

```
tw_gcal_sync -c warrior -t "DUE"
```

#### versions
tw_gcal_sync, version 1.2.1
taskwarrior 2.6.2

**Comment by bergercookie** [comment]
Not supported by `taskw` but it should be automatically fixed once we switch to `tasklib` - see #65

**Comment by bergercookie** [comment]
Hi @amiroslaw  

This should now be supported across all the apps in this repo that interact with Taskwarrior. Use the custom taskwarrior filtering in order to specify that you don't want to synchronize tags of a particular tag. To include tags by a virtual tag in the sync, you have to specify that virtual tag via a custom TW filter. For example:

```
tw_gtasks_sync  -l some_google_task_list -t tw_tag_to_include  -f "+OVERDUE"
```

This feature should work on latest master - commit https://github.com/bergercookie/syncall/commit/0893f12870aff5242118993c70f61bbfc85544cb - and should make its way to a pypi release in the coming week(s)
